beacon-chain/rpc/eth/validator/validator.go

```diff
@@ -334,6 +337,8 @@ func (vs *Server) ProduceBlockV2(ctx context.Context, req *ethpbv1.ProduceBlockR
 // ProduceBlockV2SSZ requests the beacon node to produce a valid unsigned beacon block, which can then be signed by a proposer and submitted.
 //
 // The produced block is in SSZ form.
+// By definition `/eth/v2/validator/blocks/{slot}/ssz`, does not produce block using mev-boost and relayer network:
+// https://ethereum.github.io/beacon-APIs/?urls.primaryName=v2.3.0#/Validator/produceBlockV2
```

Comment:
Please add a suggestion to which API endpoint is supposed to be used for MEV boost / relay. I.e. blinded blocks endpoint. 

The link you provided doesn't define anything about whether or not mev-boost should be used.